config/main.py

```diff
@@ -2105,7 +2105,7 @@ def add_reasons(counter_name, reasons, verbose):
 #
 # 'remove_reasons' subcommand ('config dropcounters remove_reasons')
 #
-@dropcounters.command()
+@dropcounters.command('remove_reasons')
```

Comment:
is this release as well? I suggest to change to add-reasons

Reply:
Changed.